--- imp-4.2-rc2/config/prefs.php.dist~	2008-02-07 00:59:33.000000000 +0200
+++ imp-4.2-rc2/config/prefs.php.dist	2008-02-07 01:00:22.684662278 +0200
@@ -6,7 +6,7 @@
  */
 
 // Make sure that constants are defined.
-require_once dirname(__FILE__) . '/../lib/IMP.php';
+require_once '/usr/share/horde/imp/lib/IMP.php';
 $is_pop3 = isset($_SESSION['imp']) &&
            $_SESSION['imp']['base_protocol'] == 'pop3';
 
