--- imp-h3-4.0.1/templates/login/login.inc	2004-12-20 10:41:30.000000000 +0100
+++ imp-h3-4.0.1-new/templates/login/login.inc	2005-01-29 01:00:42.336330560 +0100
@@ -123,7 +123,7 @@
 <br class="spacer" />
 <?php $notification->notify(array('listeners' => 'status')) ?>
 
-<table width="100%"><tr><td align="center"><table border="0" width="300">
+<table width="100%"><tr><td align="center"><table border="0" width="300" style="margin: auto;">
 
 <?php if (!empty($reason)): ?>
 <tr>
