--- imp-h3-4.3/lib/api.php~	2008-09-24 20:04:53.000000000 +0200
+++ imp-h3-4.3/lib/api.php	2008-10-16 18:14:29.000000000 +0200
@@ -143,7 +143,7 @@
         require IMP_BASE . '/config/servers.php';
     }
 
-    return IMP_Session::createSession($userID, $credentials['password'], $server_key, empty($result[$server_key]) ? array() : $result[$server_key]);
+    return IMP_Session::createSession($userID, $credentials['password'], $server_key, empty($servers[$server_key]) ? array() : $servers[$server_key]);
 }
 
 /**
--- imp-h3-4.3/lib/Session.php~	2008-06-30 22:11:36.000000000 +0200
+++ imp-h3-4.3/lib/Session.php	2008-10-16 20:41:42.000000000 +0200
@@ -102,9 +102,9 @@
          * environment-wide settings. Do NOT use the global $servers variable
          * as it may not exist. */
         if (is_callable(array('Horde', 'loadConfiguration'))) {
-            $result = Horde::loadConfiguration('servers.php', array('servers'));
-            if (!is_a($result, 'PEAR_Error')) {
-                extract($result);
+            $servers = Horde::loadConfiguration('servers.php', 'servers');
+            if (is_a($servers, 'PEAR_Error')) {
+                $servers = array();
             }
         } else {
             require IMP_BASE . '/config/servers.php';
@@ -190,7 +190,7 @@
 
             /* Set the maildomain. */
             $maildomain = $GLOBALS['prefs']->getValue('mail_domain');
-            $_SESSION['imp']['maildomain'] = ($maildomain) ? $maildomain : $args['maildomain'];
+            $_SESSION['imp']['maildomain'] = ($maildomain) ? $maildomain : $ptr['maildomain'];
         }
 
         /* Determine the base protocol. */
