--- imp-h3-4.0.2-rc1/templates/login/login.inc		2005-02-03 16:08:04.363519592 +0100
+++ imp-h3-4.0.2-rc1-new/templates/login/login.inc	2005-02-03 16:10:49.766374536 +0100
@@ -123,7 +123,7 @@
 <br class="spacer" />
 <?php $notification->notify(array('listeners' => 'status')) ?>
 
-<table width="100%"><tr><td align="center"><table width="300" align="center">
+<table width="100%"><tr><td align="center"><table width="300" align="center" style="margin: auto;">
 
 <?php if (!empty($reason)): ?>
 <tr>
