--- imp-4.1/lib/IMAP/Client.php~	2006-02-21 08:39:54.000000000 +0100
+++ imp-4.1/lib/IMAP/Client.php	2006-04-05 17:11:10.000000000 +0200
@@ -411,7 +411,7 @@
             return PEAR::raiseError(_("If using SSL or TLS, you must have the PHP openssl extension loaded."), 'horde.error');
         }
 
-        if ($res = $this->UseTLS()) {
+        if ($res = $this->useTLS()) {
             if (is_a($res, 'PEAR_Error')) {
                 return $res;
             } else {
