diff -uNr imp-3.2.6/config/prefs.php.dist imp-3.2.6.new/config/prefs.php.dist
--- imp-3.2.6/config/prefs.php.dist	2002-10-29 22:04:33.000000000 +0100
+++ imp-3.2.6.new/config/prefs.php.dist	2004-11-04 01:42:19.744042568 +0100
@@ -2,7 +2,7 @@
 // $Horde: imp/config/prefs.php.dist,v 1.49.2.22 2002/10/29 21:04:33 chuck Exp $
 
 // Make sure that constants are defined.
-require_once dirname(__FILE__) . '/../lib/constants.php';
+require_once '/usr/share/horde/imp/lib/constants.php';
 
 // Changes you make to this file will not be reflected until the user
 // logs out and logs in again.
